src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -179,6 +209,40 @@ export class CdkPopoverEdit<C> implements AfterViewInit, OnDestroy {
         this.viewContainerRef,
         {$implicit: this.context}));
     this.focusTrap!.focusInitialElementWhenReady();
+
+    // Update the size of the popup initially and on subsequent changes to
+    // scroll position and viewport size.
+    merge(this.scrollDispatcher.scrolled(), this.viewportRuler.change())
+        .pipe(
+            startWith(null),
+            takeUntil(this.overlayRef!.detachments()),
+            )
+        .subscribe(() => {
+          this._updateOverlaySize();
+        });
+  }
+
+  private _getOverlayCells(): HTMLElement[] {
+    const cell = closest(this.elementRef.nativeElement!, CELL_SELECTOR) as HTMLElement;
```

Comment:
Maybe we should make `closest` accept an `ElementRef` so that we don't have to repeat `.nativeElement` everywhere. There's also a `coerceElement` in `@angular/cdk/coercion` to make it a little easier.